storage-go/client/storage_client.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	pb "github.com/jadekler/dummy-benchmarking-repo/storage-go/proto"
+	"google.golang.org/grpc"
+)
+
+const address = "localhost:50051"
+
+func main() {
+	conn, err := grpc.Dial(address, grpc.WithInsecure())
+	if err != nil {
+		fmt.Printf("did not connect: %v", err)
```

Comment:
This won't exit the program - it'll continue, with a nil `conn`. We should use `log.Fatal(err)`